File: CoverPoints.swift
import Foundation

class Solution {
	func coverPoints(_ A: inout [Int], _ B: inout [Int]) -> Int {
        
        let count = A.count
        var totalSteps = 0
        for index in 0..<(count-1) {
            let deltaX = abs(A[index] - A[index+1])
            let deltaY = abs(B[index] - B[index+1])
            let steps = abs(deltaX - deltaY) + min(deltaX,deltaY)
            totalSteps += steps
        }
        return totalSteps
        
	}
}

File: CoverPoints.py
class Solution:
    # @param A : list of integers
    # @param B : list of integers
    # @return an integer
    def coverPoints(self, A, B):
        
        count = len(A)
        totalSteps = 0
        for index in range(count-1):
            deltaX = abs(A[index] - A[index+1])
            deltaY = abs(B[index] - B[index+1])
            steps = abs(deltaX - deltaY) + min(deltaX,deltaY)
            totalSteps += steps
        return totalSteps
        